robot: ibuki
dofs: 50
rate: 40
broadcast: 5
joint_mask_h: 33554431
joint_mask_l: 33554431
weight_hw_adjust: 0
weight_idle: 1
weight_reflex: 1 
weight_slave: 1
weight_auto: 1
